### DISCUSSION TITLE
[General] Never animate modules by default

### DISCUSSION THREAD

**Discussion by HorstBaerbel** [vcync/modV#896] [Ideas]
**Is your feature request related to a problem? Please describe.**
This is a conceptual thing. It would break existing modules and should probably go to the docs too.
My view on things is that absolutely no module should be animated by default. This only takes away options from the module user. Modules can be easily animated by tweeing values, using audio or MIDI input, so it is better to expose a specific value to the UI than to hardwire it to e.g. "time/2".

**Describe the solution you'd like**
Remove all animation from all modules and add a statement to the docs for new module creators

**Comment by 2xAA** [comment]
I'll move this to a discussion as it's conceptual
